packages/graphql/src/graphql/input-objects/generic-operators/DateScalarFilters.ts

```diff
@@ -34,3 +34,12 @@ export const DateScalarFilters = new GraphQLInputObjectType({
         lessThanEquals: { type: GraphQLDate },
     },
 });
+
+export const DateListFilters = new GraphQLInputObjectType({
```

Comment:
As the difference between these ListFilters is only the type, maybe we can generate them rather than write them manually. But, I'm not suggesting, just a thought!

Reply:
I though about that, but realistically, we would still need to call the generators with the relevant information, so while it makes it a bit less verbose it won't be for much without adding complexity